content/en/tracing/trace_collection/trace_context_propagation/java.md

```diff
@@ -27,16 +27,16 @@ Injection styles can be configured using:
 - System Property: `-Ddd.trace.propagation.style.inject=datadog,b3multi`
 - Environment Variable: `DD_TRACE_PROPAGATION_STYLE_INJECT=datadog,b3multi`
 
-The value of the property or environment variable is a comma (or space) separated list of header styles that are enabled for injection. By default only Datadog injection style is enabled.
+The value of the property or environment variable is a comma (or space) separated list of header styles that are enabled for injection. By default the `datadog,tracecontext` injection styles are enabled.
```

Comment:
```suggestion
The value of the property or environment variable is a comma (or space) separated list of header styles that are enabled for injection. By default, the `datadog,tracecontext` injection styles are enabled.
```
Can the two styles be separated? If they can, I recommend changing the last sentence to:
By default, the `datadog` and `tracecontext` injection styles are enabled.

Reply:
I use `datadog,tracecontext` format because it is the expected format for the value parameter and it is used in all other languages like [dotnet](https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/trace_context_propagation/dotnet.md), [go](https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/trace_context_propagation/go.md) and [Python](https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/trace_context_propagation/python.md).

Weirdly, it is not for [PHP]( https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/trace_context_propagation/php.md) and [Ruby](https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/trace_context_propagation/ruby.md) -- I just discover it. I will use you proposed style then 👍

Reply:
A concern I have about this change is (as I am not native English speaker), is it clear for the reader that `datadog` and `tracestate` means `datadog` first (it has priority over) `tracestate` in second?

Reply:
If it's important that the `datadog` style takes precedence over the `tracecontext` style, it's clearest to state the difference in priority in a separate sentence. You could add something like, "The `datadog` style` has higher priority than the `tracestate` style."

Reply:
Great thanks! I update the sentence. It should be good 👍